src/poliastro/twobody/orbit.py

```diff
@@ -749,48 +749,90 @@ def circular(
         )
 
     @classmethod
-    @u.quantity_input(angular_velocity=u.rad / u.s, period=u.s, hill_radius=u.m)
-    def geostationary(
-        cls, attractor, angular_velocity=None, period=None, hill_radius=None
-    ):
-        """Return the geostationary orbit for the given attractor and its rotational speed.
+    def stationary(cls, attractor):
+        """Return the stationary orbit for the given attractor and its rotational speed.
 
         Parameters
         ----------
         attractor : Body
             Main attractor.
-        angular_velocity : ~astropy.units.Quantity
-            Rotational angular velocity of the attractor.
-        period : ~astropy.units.Quantity
-            Attractor's rotational period, ignored if angular_velocity is passed.
-        hill_radius : ~astropy.units.Quantity
-            Radius of Hill sphere of the attractor (optional). Hill sphere radius(in
-            contrast with Laplace's SOI) is used here to validate the stability of the
-            geostationary orbit, that is to make sure that the orbital radius required
-            for the geostationary orbit is not outside of the gravitational sphere of
-            influence of the attractor.
-            Hill SOI of parent(if exists) of the attractor is ignored if hill_radius is not provided.
+
         """
+        return cls.synchronous(attractor)
 
-        if angular_velocity is None and period is None:
-            raise ValueError(
-                "At least one among angular_velocity or period must be passed"
-            )
+    @classmethod
+    @u.quantity_input(
+        a=u.km,
+        ecc=u.one,
+        inc=u.deg,
+        argp=u.deg,
+        arglat=u.deg,
+        raan=u.deg,
+        period_mul=u.one,
+    )
+    def synchronous(
+        cls,
+        attractor,
+        a=None,
+        ecc=0 * u.one,
+        inc=0 * u.deg,
+        argp=0 * u.deg,
+        arglat=0 * u.deg,
+        raan=0 * u.deg,
+        epoch=J2000,
+        plane=Planes.EARTH_EQUATOR,
+        period_mul=1 * u.one,
```

Comment:
I envision users changing this parameter more often than the rest of the elements, perhaps we can place it after `attractor`